### PR TITLE
More memory to solve PermGen space problem.

### PR DESCRIPTION
**Pull request by rohitsinha54**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -212,6 +212,9 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.14.1</version>
+        <configuration>
+          <argLine>-Xmx2048m -XX:MaxPermSize=256m -XX:+CMSClassUnloadingEnabled</argLine>
```
Comment (abaranau) [inline pom.xml]: t-rex already asked about it: do we need -XX:+CMSClassUnloadingEnabled ?

Comment (rohitsinha54) [inline pom.xml]: We faced the same issue in Spark example inside CDAP repo. Without class unloading they pass but sometime goes out of  mem. Hence, we have them in our CDAP example pom

**Comment by rohitsinha54** [comment]
Build passed here: https://builds.cask.co/browse/CG-SGUT2-2

**Comment by chtyim** [comment]
LGTM

**Pull request merged**